qemu/tests/pci_devices.py

```diff
@@ -297,6 +297,8 @@ def run(test, params, env):
     env_process.preprocess_vm(test, params, env, params["main_vm"])
     vm = env.get_vm(params["main_vm"])
     qdev = vm.make_create_command()    # parse params into qdev
+    if isinstance(qdev, tuple):
```

Comment:
It changed recently because of tp-spice and yes, `make_create_command` is internal to `qemu` provider so it should be fine. Although some tests were broken because of this...